lib/Sema/TypeCheckDecl.cpp

```diff
@@ -688,8 +688,16 @@ bool
 ExistentialConformsToSelfRequest::evaluate(Evaluator &evaluator,
                                            ProtocolDecl *decl) const {
   // Marker protocols always self-conform.
-  if (decl->isMarkerProtocol())
+  if (decl->isMarkerProtocol()) {
+    // Except for BitwiseCopyable an existential of which is non-trivial.
+    auto *bitwiseCopyableProtocol =
+        decl->getASTContext().getProtocol(KnownProtocolKind::BitwiseCopyable);
```

Comment:
This will load the protocol. We'll probably end up doing it anyway when we synthesize conformances, but generally it's better to do `if (decl->getKnownProtocolKind() == ...) { ... }`

Reply:
https://github.com/apple/swift/pull/71468